app/Http/Controllers/API/ConferencesController.php

```diff
@@ -7,24 +7,17 @@
 use Carbon\Carbon;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Auth;
+use App\Http\Requests\StoreConferenceRequest;
 
 class ConferencesController extends Controller
 {
-    public function store(Request $request)
+    public function store(StoreConferenceRequest $request)
     {
-        $after = (new Carbon($request->input('start_date')))->subDay();
```

Comment:
Also, Michael agrees with this 

https://twitter.com/michaeldyrynda/status/759091692230234112

@michaeldyrynda ☝️ 😄